components/EmailValidator.vue

```diff
@@ -78,6 +80,10 @@ export default {
       required: false,
       default: '',
     },
+    validateDomain: {
```

Comment:
This parameter is currently never used, right?  In that case please remove it - I prefer to avoid adding code for future use-cases unless they are definitely happening soon, as it can lead to tech debt.